src/lib/pickles/composition_types/composition_types.ml

```diff
@@ -7,6 +7,8 @@ module Spec = Spec
 module Opt = Plonk_types.Opt
 open Core_kernel
 
+[@@@warning "-45"]
```

Comment:
Do we want this to be off globally? It would also be nice to add a comment to these mentioning what e.g. 45 represents.

Reply:
You pushed me to put it on the functions where it was needed. Thanks!